pkgs/by-name/pm/pmbootstrap/package.nix

```diff
@@ -26,6 +26,10 @@ python3Packages.buildPythonApplication rec {
   # Tests depend on sudo
   doCheck = stdenv.isLinux;
 
+  build-system = [
+    python3Packages.setuptools
+  ];
+
   nativeCheckInputs = [
     python3Packages.pytestCheckHook
     python3Packages.setuptools
```

Comment:
do we need setuptools in here after adding it to build-system?

Reply:
@onny we don't, thanks for that